charts/prefect-agent/templates/deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
       {{- end }}
       labels: {{- include "common.labels.standard" . | nindent 8 }}
         app.kubernetes.io/component: agent
+        prefect-version: {{ .Chart.AppVersion }}
```

Comment:
I guess this isn't included in the common labels?

Reply:
correct!

Reply:
https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_labels.tpl